digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/integration/Viite_13_218_spec.scala

```diff
@@ -181,14 +180,14 @@ class Viite_13_218_spec extends FunSuite with Matchers with BeforeAndAfter {
         f.setAccessible(true)
         var x = f.get(c)
         x = x match {
-          case option: Option[AnyRef] if option.isDefined => f.get(c).asInstanceOf[Option[AnyRef]].get
+          case Some => f.get(c).asInstanceOf[Option[AnyRef]].get
```

Comment:
👌 Jes, siistimpää koodia!

Heitä tästä joku "---and scalafying old code" -lisäkommentti commitviestiin ton varsinaisen hovertekstipihvin lisäks, niin mahdollinen commitin myöhempi tutkija ihmettelee vähemmän.